config/core/test-configs.yaml

```diff
@@ -3360,9 +3369,7 @@ test_configs:
       - v4l2-decoder-conformance-h264
       - v4l2-decoder-conformance-vp8
       - v4l2-decoder-conformance-vp9
-    filters: &chromebook-decoders-filter
-      - passlist: {defconfig: ['videodec']}
-      - regex: {'tree': '(next|mainline|media)'}
+    filters: *chromebook-decoders-filter
```

Comment:
This is not related to asurada but to kevin, so maybe it should be split out into a separate commit.

Reply:
I just moved the filter definition up since `asurada` comes before `kevin` alphabetically, there's no functional change for `kevin`. I'm happy to split the changes into two commits anyway if it makes things cleaner!

Reply:
Ah sorry I see that now, it's fine then.  OK I think this is ready to be merged :+1: